### PR TITLE
porting: configuring: DeviceInfo: add Mir section

### PR DESCRIPTION
**Pull request by amartinz**
See: https://gitlab.com/ubports/development/core/hybris-support/mir-android-platform/-/merge_requests/34

### CODE REVIEW

File: porting/configure_test_fix/device_info/Mir.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+mir-platform-android
+--------------------
+
+======================================  ===========  =============================================  ===========================
+Key                                     Description  Value(s)                                       Default value
+======================================  ===========  =============================================  ===========================
+MirAndroidPlatformClientEglFlush        See below    Comma-separated list (or special value "all")  Empty
+MirAndroidPlatformClientFenceSync       See below    Comma-separated list (or special value "all")  Empty
+MirAndroidPlatformServerBackpressure    See below    Boolean                                        False
+MirAndroidPlatformServerEglSync         See below    Boolean                                        Android property (``TODO``)
```
Comment (amartinz) [inline porting/configure_test_fix/device_info/Mir.rst:24]: TODO: add android property which gets overruled.

File: porting/configure_test_fix/device_info/Mir.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Allows redrawing to be skipped or not after a missed frame, similarly to SurfaceFlinger.
+
+MirAndroidPlatformServerEglSync
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Allow or avoid synchronization using EGL fencing primitives when drawing the screen.
+
+Overrules existing Android property (``TODO``).
```
Comment (amartinz) [inline porting/configure_test_fix/device_info/Mir.rst:48]: TODO: add android property which gets overruled.

File: porting/configure_test_fix/device_info/Mir.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+MirAndroidPlatformServerPartialUpdates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Allow partial redraws to HWC-assigned server-side buffers.
+
+Examples
+--------
+
+TODO
```
Comment (amartinz) [inline porting/configure_test_fix/device_info/Mir.rst:58]: TODO: add example like in repowerd

**Comment by amartinz** [comment]
Also need to add new properties which got added to the MR as well as document the comma separated list properly (eg that inversion is possible, how to turn it from an allow-list into a deny-list, ...)

**Comment by amartinz** [comment]
> Also need to add new properties which got added to the MR as well as document the comma separated list properly (eg that inversion is possible, how to turn it from an allow-list into a deny-list, ...)

Done

**Review by Flohack74 — APPROVED** [review]

**Pull request merged**